internal/lib/lib.go

```diff
@@ -440,7 +440,7 @@ func GetVipNetworkList() ([]string, error) {
 	}
 	if len(vipNetworkList) > 1 {
 		// Only AWS cloud supports multiple VIP networks
-		if !(IsPublicCloud() && GetCloudType() != CLOUD_AWS) {
+		if !(IsPublicCloud() && GetCloudType() == CLOUD_AWS) {
```

Comment:
why are we checking for IsPublicCloud if AWS is guaranteed to be a public cloud?